scripts/cleanup_old_vms.py

```diff
@@ -239,10 +279,20 @@ def cleanup_vms(texts, max_hours=24, providers=None, prompt=True):
     for thread in thread_queue:
         thread.join()
 
+    with open(args.outfile, 'w') as report:
+        report.write('## VM/Instances deleted via:\n'
+                     '##   text matches: {}\n'
+                     '##   age matches: {}\n'
+                     .format(texts, max_hours))
+        message = tabulate(deleted_vms_list,
+                           headers=['Provider', 'Name', 'Age', 'Status', 'Delete RC'],
+                           tablefmt='orgtbl')
+        report.write(message)
+    print(message)
+
     print("Deleting finished")
 
 
 if __name__ == "__main__":
     args = parse_cmd_line()
-    sys.exit(cleanup_vms(args.text_to_match, args.max_hours,
-                         args.providers, args.prompt))
+    sys.exit(cleanup_vms(args.text_to_match, args.max_hours, args.providers, args.prompt))
```

Comment:
I suppose cleanup_vms should always return something If you pass this procedure results to sys.exit.
0 - success
1-n error